src/MinimalDelegation.sol

```diff
@@ -151,26 +151,31 @@ contract MinimalDelegation is
 
         Settings settings = keySettings[keyHash];
         if (settings.isExpired()) revert IKeyManagement.KeyExpired();
```

Comment:
Can isExpired return both the validity and the expiration? also we can then revert with the expiration timestamp which would be nice

and then you don't need the next line :)